eclair-core/src/main/resources/reference.conf

```diff
@@ -163,6 +163,20 @@ eclair {
     port = 9051
     private-key-file = "tor.dat"
   }
+
+  postgre = {
+    connectionPool = "HikariCP"
+    dataSourceClass = "org.postgresql.ds.PGSimpleDataSource"
```

Comment:
`PGSimpleDataSource` should not be used in production: https://jdbc.postgresql.org/documentation/publicapi/org/postgresql/ds/PGPoolingDataSource.html

Reply:
Can you elaborate on risks of using `PGSimpleDataSource`. Doc over there has a red warning, I'm not sure I understand what it means.